mlflow/config/__init__.py

```diff
@@ -43,6 +46,15 @@ def enable_async_logging(enable=True):
     MLFLOW_ENABLE_ASYNC_LOGGING.set(enable)
 
 
+def set_async_logging_threadpool_size(num_workers):
```

Comment:
I'm not sure if we need a function for this. Can we just set `MLFLOW_ASYNC_LOGGING_THREADPOOL_SIZE`?

Reply:
it's for visibility, otherwise users won't know about this. But I do think your concern makes sense! it's odd to have `threadpool_size` in the name, let me remove this function.